src/core/ColorPicker/ColorSwatch.tsx

```diff
@@ -31,7 +31,9 @@ export const ColorSwatch = React.forwardRef<HTMLDivElement, ColorSwatchProps>(
     useTheme();
 
     const c =
-      typeof color === 'string' ? color : getColorValue(color).toHexString();
+      typeof color === 'string'
+        ? color
+        : getColorValue(color).toHslString(true);
```

Comment:
why changed?

Reply:
.toHexString() doesn't use the alpha value. I guess I could use .toHexString(true) to use alpha value but I figured hslString is the same thing.